yarn.lock

```diff
@@ -2607,6 +2592,7 @@ __metadata:
     inquirer: "npm:^9.1.4"
     interpret: "npm:^3.1.1"
     liftoff: "npm:^5.0.0"
+    mypa-google-docs-importer: "npm:^1.0.0"
```

Comment:
1.0.0 is a not-working release so if possible I would set this to 1.0.1 or higher, even though it will probably get the newest always anyway.